docs/learn/learn-staking-faq.md

```diff
@@ -0,0 +1,68 @@
+
+## 2. When do I get slashed?
+
+Slashing will happen if a validator misbehaves (e.g. goes offline, attacks the network, or runs modified software) in the network. They and their nominators will get slashed by losing a percentage of their staked DOT. By choosing to [nominate reliable validators](https://support.polkadot.network/support/solutions/articles/65000169507-the-role-responsibilities-of-a-polkadot-nominator), you can minimize the risk of slashing and earn rewards for securing the network. Learn more about slashing [here](learn-staking.md#slashing)
+
+## 3. Why am I not receiving staking rewards?
+
+If you do not see staking rewards on your account, it could be due to any of the below reasons.
+
+### Delayed staking reward payout
+
+The [staking reward distribution](learn-simple-payouts) is not automatic and needs to be triggered through a transaction. Typically, it is the validators who trigger the staking reward payout transaction, but for some reason if they do not, you can claim the rewards on your own. 
+
+### Nominated validators not in the active set
+
+If none of the validators you nominated made it to the active set in an era, you will not earn any rewards for that era. To maximize your chance of earning staking rewards on every era, consider nominating multiple validators (up to 16 on Polkadot).
+
+### Nominated a validator with 100% commission
+
+When you nominate a validator who takes 100% commission on staking rewards, you will not receive any staking reward for the eras when your nomination is applied to that specific validator.
+
+### Nominated an oversubscribed validator
+
+Only the top 256 nominators of a Validator receive the staking rewards. If your nomination is applied to an oversubscribed Validator and your stake does not put you in the top 256, you will not earn rewards. On a regular basis, ensure that the Validators you chose are not oversubscribed.
+
+### Min-Active Nomination Threshold is not met
+
+With the [bags-list implementation](https://github.com/paritytech/substrate/pull/9507), the minimum stake threshold to earn staking rewards can change with each era. Do not confuse this with min-intention-threshold (10 DOT) which is required to submit your intention to nominate. The network supports 50,000 nominator intentions, but can reward only up to 22500 nominators in any given era. If you nominate with, say 50 DOT, you are not guaranteed to be part of the top 22500 and hence you may not earn staking rewards. The entry barrier to be an electing nominator is now determined dynamically, based on market demand.
```

Comment:
```suggestion
With the [bags-list implementation](https://github.com/paritytech/substrate/pull/9507), the minimum stake threshold to earn staking rewards can change with each era. Do not confuse this with min-intention-threshold (10 DOT) which is required to submit your intention to nominate. The network supports 50,000 nominator intentions, but can reward at most 22500 nominators in any given era. If you nominate with, say 50 DOT, you are not guaranteed to be part of the top 22500 and hence you may not earn staking rewards. The entry barrier to be an electing nominator is now determined dynamically, based on market demand.
```

The UI will soon hopefully have a min-electing or min-active threshold which will essentially be the lowest staked nominator who is earning. We can then advise that nominators watch this number and target a value above it to increase likelihood of making it into the next election